test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -612,11 +612,746 @@ public void EntryPointExecGraphCommand()
+                            'Data': '$output_data3',
+                            'Features': [
+                                'age',
+                                'education'
+                            ]
+                        }},
+                        'Name': 'Transforms.FeatureCombiner',
+                        'Outputs': {{
+                            'Model': '$output_model4',
+                            'OutputData': '$output_data4'
+                        }}
+                    }},
+                    {{
+                        'Inputs': {{
+                            'MaximumNumberOfIterations': 1,
+                            'NumThreads': 1,
+                            'TrainingData': '$output_data4'
+                        }},
+                        'Name': 'Trainers.LogisticRegressionBinaryClassifier',
+                        'Outputs': {{
+                            'PredictorModel': '$predictor_model'
+                        }}
+                    }},
+                    {{
+                        'Inputs': {{
+                            'PredictorModel': '$predictor_model',
+                            'TransformModels': [
+                                '$output_model1',
+                                '$output_model2',
+                                '$output_model3',
+                                '$output_model4'
+                            ]
+                        }},
+                        'Name': 'Transforms.ManyHeterogeneousModelCombiner',
+                        'Outputs': {{
+                            'PredictorModel': '$output_model'
+                        }}
+                    }},
+                    {{
+                      'Name': 'Transforms.PermutationFeatureImportance',
+                      'Inputs': {{
+                        'Data': '$data',
+                        'PredictorModel': '$output_model',
+                        'PermutationCount': 5
+                      }},
+                      'Outputs': {{
+                        'Metrics': '$output_data'
+                      }}
+                    }}
+                  ],
+                  'Outputs': {{
+                    'output_data': '{1}'
+                  }}
+                }}", EscapePath(inputDataPath), EscapePath(outputDataPath));
+
+            var jsonPath = DeleteOutputPath("graph.json");
+            File.WriteAllLines(jsonPath, new[] { inputGraph });
+
+            var args = new ExecuteGraphCommand.Arguments() { GraphPath = jsonPath };
+            var cmd = new ExecuteGraphCommand(Env, args);
+            cmd.Run();
+
+            var mlContext = new MLContext();
+            var loadedData = mlContext.Data.LoadFromBinary(outputDataPath);
+            Assert.NotNull(loadedData.Schema.GetColumnOrNull("FeatureName"));
+            Assert.NotNull(loadedData.Schema.GetColumnOrNull("AreaUnderRocCurve"));
+            Assert.NotNull(loadedData.Schema.GetColumnOrNull("Accuracy"));
```

Comment:
Do we want to verify that the numbers that come back here are correct?

Reply:
We would verify this by loading the same predictor model and data that goes into the PFI node, and applying the same steps to them that the PFI node does, so I don't see the point of doing this.